qfieldsync/gui/package_dialog.py

```diff
@@ -189,6 +192,8 @@ def package_project(self):
         try:
             QApplication.setOverrideCursor(Qt.WaitCursor)
             offline_convertor.convert()
+            self.project.setTitle(self.original_title_project)
+            open_project(self.project.fileName())
```

Comment:
I guess you don't need it in this PR, do you?

Reply:
Due this was removed on the other [PR](https://github.com/opengisch/qfieldsync/pull/616) now this will be necessary there